src/app/filelogger.cpp

```diff
@@ -49,7 +49,7 @@ FileLogger::FileLogger(const QString &path, const bool backup, const int maxSize
         this->deleteOld(age, ageType);
 
     const Logger *const logger = Logger::instance();
-    for (const Log::Msg &msg : asConst(logger->getMessages()))
+    for (const Log::Msg &msg : logger->getMessages())
```

Comment:
Why is this unrelated change?

Reply:
@fusioneer-GitHub
You (probably) misunderstood of my [previous comment](https://github.com/qbittorrent/qBittorrent/pull/10527#discussion_r287023829), I meant you should only remove it in the code you modified/added not the whole .cpp file.